src/main/java/uk/gov/ons/ctp/response/action/export/domain/PrintFilesInfo.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.ons.ctp.response.action.export.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class PrintFilesInfo {
```

Comment:
Really minor one, but do you think it should be '_PrintFile_' not `PrintFiles` because each instance of this class relates only to one printfile, not printfiles plural?

Reply:
fixed